crates/consensus/beacon/src/engine/mod.rs

```diff
@@ -316,6 +333,7 @@ where
 
         // Set the next pipeline state.
         loop {
+            tracing::info!("processing msgs");
```

Comment:
is the info target inside a loop reasonable?

Reply:
removed this, idk how it got into this PR. maybe some staged changes locally